prime-dotnet-webapi/Models/Certifications/CollegeLicenseGrouping.cs

```diff
@@ -0,0 +1,14 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Prime.Models
+{
+    [Table("CollegeLicenseGrouping")]
+    public class CollegeLicenseGrouping
```

Comment:
should probably implement ILookup<int>
Also, table name should probably be CollegeLicenseGrouplingLookup to be consistent with our naming convention